src/AppInstallerCLICore/Commands/SearchCommand.cpp

```diff
@@ -75,11 +76,26 @@ namespace AppInstaller::CLI
     {
         context.SetFlags(Execution::ContextFlag::TreatSourceFailuresAsWarning);
 
-        context <<
-            Workflow::OpenSource() <<
-            Workflow::SearchSourceForMany <<
-            Workflow::HandleSearchResultFailures <<
-            Workflow::EnsureMatchesFromSearchResult(false) <<
-            Workflow::ReportSearchResult;
+
+        if (context.Args.Contains(Execution::Args::Type::ListVersions))
+        {
+            context <<
+                Workflow::OpenSource() <<
+                Workflow::SearchSourceForMany <<
+                Workflow::HandleSearchResultFailures <<

```

Comment:
nit: we should move common workflows above the if else block

          context <<
                Workflow::OpenSource() <<
                Workflow::SearchSourceForMany <<
                Workflow::HandleSearchResultFailures <<

Reply:
Moved